app/services/index.ts

```diff
@@ -1,5 +1,7 @@
 import todosService from './todos';
+import cardsService from './cards';
 
 export default {
-  todosService
+  todosService,
```

Comment:
maybe ending the variable in service is redundant?

Reply:
if the variable cardsService is not called when want to test in npm run console it will not work